src/main/java/seedu/address/logic/parser/FindCommandParser.java

```diff
@@ -38,25 +39,42 @@ public FindCommand parse(String args) throws ParseException {
                     String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
         }
 
-        if (argMultimap.getValue(PREFIX_NAME).isPresent() && argMultimap.getValue(PREFIX_PHONE).isPresent()) {
+        if (argMultimap.getValue(PREFIX_NAME).isPresent() && argMultimap.getValue(PREFIX_PHONE).isPresent()
+            && argMultimap.getValue(PREFIX_EMAIL).isEmpty() && argMultimap.getValue(PREFIX_TAG).isEmpty()
```

Comment:
Is it possible to abtract the if conditions to another method? So it looks less cluttered